### ISSUE TITLE
Error with KSampler:  izes of tensors must match except in dimension 1. Expected size 768 but got size 1024 for tensor number 1 in the list.

### ISSUE THREAD

**Issue by qingyuan18** [ExponentialML/ComfyUI_Native_DynamiCrafter#23]
comfyui version:  
ComfyUI: 2228[56333d](https://2024-06-07)
Manager: V2.36

DynamicCraft version: 2024-05-23

workflow :sample workflow in this github

error: 
Error occurred when executing KSampler:

Sizes of tensors must match except in dimension 1. Expected size 768 but got size 1024 for tensor number 1 in the list.

"/home/ubuntu/ComfyUI/custom_nodes/ComfyUI_Native_DynamiCrafter/lvdm/modules/networks/openaimodel3d.py", line 703, in forward
context = context_processor(context, num_video_frames, img_emb=img_emb)
File "/home/ubuntu/ComfyUI/custom_nodes/ComfyUI_Native_DynamiCrafter/lvdm/modules/networks/openaimodel3d.py", line 327, in context_processor
context = torch.cat([context, img_emb.to(context.device, context.dtype)], dim=1)

<img width="982" alt="image" src="https://github.com/ExponentialML/ComfyUI_Native_DynamiCrafter/assets/35717759/bded20fe-e1c9-4b44-a29b-8a8e62566f36">

**Comment by qingyuan18** [comment]
hi team
   any update from this issue?
